From 395dd9256a785a9d318ffa443b1a12dfa69ab22c Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 13 Oct 2021 14:43:51 -0700
Subject: [PATCH 08/45] portenta-m8: add dfu_alt_info environment var

This var specifies how to handle a boot software update via
EFI capsule update.

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/portenta-m8.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/portenta-m8.h b/include/configs/portenta-m8.h
index 0153e7dfb3..7ff4f8a30c 100644
--- a/include/configs/portenta-m8.h
+++ b/include/configs/portenta-m8.h
@@ -151,6 +151,8 @@
 	"fdt_addr=0x43000000\0"			\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fit=no\0" \
+	"dfu_alt_info=mmc 2=imx-boot raw 0x42 0x250 mmcpart 1;" \
+		"u-boot-itb raw 0x300 0xD00 mmcpart 1\0" \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"bootm_size=0x10000000\0" \
-- 
2.35.1

